src/components/Index.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import '../styles/sign.css'
+import SignIn from "./SignIn";
+import SignUp from "./SignUp";
+
+
+const Index = () => {
+    const [sign, setSign] = useState(false);
+
+
+    return (
+        <div className="container">
+
+            <div className="text">
+                <h1>Facebook</h1>
+                <p>Facebook helps you connect and share with the people in your life.</p>
+            </div>
+            <div className="sign">
+                <SignIn setSign={setSign} />
+                {sign && <SignUp setSign={setSign} />}
```

Comment:
you should change that to have the button that toggles between the ``` signUp ``` & ``` signIn ``` components to be in the index component itself and in that case you will not need to pass the ``` setSign ``` as a proops.
And after doing that this can be replaced with a ``` ternary operator ``` to choose which component to render,
and it will be something like that ``` {signUpView? <SignUp/> : <SignIn} ```